### PR TITLE
Convert to Vite React App. Add copy and basic styling for landing page.

### PR DESCRIPTION
**Pull request by tusharhchhabra**

### CODE REVIEW

File: src/App.css
Comment (MarKuchar) [inline src/App.css:1]: This file can be deleted.

**Review by MarKuchar — COMMENTED** [review]
It looks good overall 👍, just a couple of updates to consider.

File: vite.config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
```
Comment (MarKuchar) [inline vite.config.js:6]: We could add
`  server: {
    port: 3000,
  },`
here so we keep the port 3000.

Comment (tusharhchhabra) [reply vite.config.js:6]: Need the 3000 for my bootcamp projects unfortunately (multiple of them use it). For Vite projects let's keep it 5173?

**Review by MarKuchar — APPROVED** [review]

**Pull request merged**